### ISSUE TITLE
Question: Is it possible to run server in cluster mode like in node.js?

### ISSUE THREAD

**Issue by devalexqt** [aaronwlee/Attain#15]
Hi!
Do we have any alternative in deno to run server on all available CPUs in parallel, like cluster/pm2 module in node.js?

**Comment by aaronwlee** [comment]
I couldn't find that yet... 
even if they have, I need documentation for proper use cases and usages.

**Comment by devalexqt** [comment]
This is similar to a load balancer that shares a port for multiple instances of a server-side script.

**Comment by aaronwlee** [comment]
If you really need it for now, then I recommend using the Kubernetes

**Comment by devalexqt** [comment]
Thanks, not now but for future.

**Comment by devalexqt** [comment]
https://github.com/denoland/deno/issues/5965
